DfE.FindInformationAcademiesTrusts/Pages/Trusts/Overview.cshtml

```diff
@@ -0,0 +1,97 @@
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Pupil numbers
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  4,896
+                </dd>
+              </div>
+              <div class="govuk-summary-list__row">
+                <dt class="govuk-summary-list__key govuk-!-width-one-half">
+                  Pupil capacity <br>(% full)
+                </dt>
+                <dd class="govuk-summary-list__value">
+                  6,463<br>(76%)
+                </dd>
+              </div>
+            </dl>
+          </div>
+        </section>
+      </div>
+      <div class="govuk-grid-column-one-half">
+        <section class="govuk-summary-card" aria-labelledby="ofsted-ratings">
+          <div class="govuk-summary-card__title-wrapper">
+            <h2 class="govuk-summary-card__title" id="ofsted-ratings">Ofsted ratings</h2>
+          </div>
+          <div class="govuk-summary-card__content">
+            <table class="govuk-table" data-module="moj-sortable-table">
+              <thead class="govuk-table__head">
+                <tr class="govuk-table__row">
+                  <th scope="col" class="govuk-table__header" aria-sort="ascending">Rating</th>
+                  <th scope="col" class="govuk-table__header govuk-table__header--numeric govuk-!-width-one-third" aria-sort="none">Number of schools</th>
+                </tr>
+              </thead>
+              <tbody class="govuk-table__body">
+                <tr class="govuk-table__row">
+                  <td class="govuk-table__cell" data-sort-value="1">Outstanding</td>
+                  <td class="govuk-table__cell govuk-table__cell--numeric">3</td>
+                </tr>
+                <tr class="govuk-table__row">
+                  <td class="govuk-table__cell" data-sort-value="2">Good</td>
+                  <td class="govuk-table__cell govuk-table__cell--numeric">5</td>
+                </tr>
+                <tr class="govuk-table__row">
+                  <td class="govuk-table__cell" data-sort-value="3">Requires improvement</td>
+                  <td class="govuk-table__cell govuk-table__cell--numeric">2</td>
+                </tr>
+                <tr class="govuk-table__row">
+                  <td class="govuk-table__cell" data-sort-value="4">Inadequate</td>
+                  <td class="govuk-table__cell govuk-table__cell--numeric">3</td>
+                </tr>
+                <tr class="govuk-table__row">
+                  <td class="govuk-table__cell" data-sort-value="5">Not yet inspected</td>
+                  <td class="govuk-table__cell govuk-table__cell--numeric">4</td>
+                </tr>
+              </tbody>
+            </table>
+            <div aria-live="polite" role="status" aria-atomic="true" class="govuk-visually-hidden"></div>
```

Comment:
What is this for?

Reply:
This was copied over from the prototype but I don't think its doing anything right now